README.md

```diff
@@ -44,8 +44,8 @@ Select *Git* then configure:
 Select *Stash Pull Request Builder* then configure:
 
 - **Cron**: must be specified. eg: every 2 minute `H/2 * * * *`
-- **Stash Host**: the *http* or *https* URL of the Stash host (NOT *ssh*). eg: *https://example.com*
-- **Stash Credentials**: Select or Add the login username/password for the Stash Host
+- **Stash REST API Host (HTTP URL)**: the *http* or *https* URL of the Stash host (NOT *ssh*). eg: *https://example.com*
```

Comment:
Why not just `Stash URL: the *http* or *https* URL of the Stash instance (NOT *ssh*). eg: *https://example.com*`

Reply:
Not sure in hindsight, but probably made sense to differentiate the REST API aspect from just the git protocol server that can be accessed over http(s) or ssh :)